### ISSUE TITLE
input step conflict

### ISSUE THREAD

**Issue by fabiulous** [omichelsen/angular-steps#3]
If you add a number input with step, it conflicts with your directive.

```html
<form>
  <input type="number" name="points" min="0" max="100" step="10" value="30">
</form>
```

Could you check it out?

**Comment by gggordon** [comment]
The issue is not related to the input type but the step attribute as the directive binds to both elements and attributes as defined on https://github.com/omichelsen/angular-steps/blob/master/src/angular-steps.js#L20

**Comment by omichelsen** [comment]
I think a fix would be to remove the "A" as accepted use, however this would be a major/breaking change.

```
angular.module('angular-steps').directive('step', function () {
  return {
    restrict: 'E',
```

**Comment by gggordon** [comment]
That's a good idea. Also, adding a vendor prefix, eg `ng-omi-step` to components may prevent further conflicts with other packages or future enhancements. 

```
angular.module('angular-steps').directive('ngOmiStep', function () {
  return {
    restrict: 'EA',
```

**Comment by omichelsen** [comment]
I have released version v1.0.0 which disallows attribute use of directives.
